supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc

```diff
@@ -0,0 +1,245 @@
+[[red-hat-directory-server-conventions]]
+
+For documentation questions, contact rhel-docs@redhat.com.
+
+[discrete]
+[[access-control-instruction]]
+==== image:images/yes.png[yes] access control instruction (noun)
+*Description*: In an LDAP directory, an access control instruction (ACI) grants or denies access to entries or attributes. Use access control instruction on the first usage and then the abbreviation ACI subsequently.
```

Comment:
Suggest putting "access control instruction" and "ACI" in quotes in the second sentence.

Reply:
Fixed